models/library.py

```diff
@@ -49,7 +49,8 @@ class RentalLog(db.Model):
                         nullable=False)
     _borrow_time = db.Column(db.DateTime)
     _return_time = db.Column(db.DateTime)
-    reserved = db.Column(db.Boolean)
+    reservation_status = db.Column(db.Boolean)
```

Comment:
I think it should't be Boolean column. Later we should use ChoiceType:
http://sqlalchemy-utils.readthedocs.io/en/latest/data_types.html

Reply:
Ok. I already started to change it after our Daily. Now I know that it should be Enum. I